doc/configuration/instrumentation.md

```diff
@@ -29,6 +29,9 @@ General options for the Exometer reporter:
 * **Default:** `false`
 * **Example:** `all_metrics_are_global = true`
 
+When enabled, all per-host metrics are merged into global equivalents.
+This option significantly reduces CPU and (especially) memory footprint in setups with exceptionally many domains (thousands, tens of thousands).
```

Comment:
Actually, this option aggregates all per-host-**type** metrics into global ones. Even if you have thousands of domains, it has no effect if they are dynamic, because you would still have one host type. The option should be used only if you have many host types or static hosts (static domains). Maybe you could rephrase this for better clarification? By the way, I remember testing metrics with 512 static hosts, and this used to be way too many already, so the option would be used sooner than that. I'd say - for hundreds of static hosts (or host types) - you need this option.

Reply:
Ah, right, thanks for clarification. I copied the old text and thought it applies here as well.

Reply:
Changed, I think it's better now.